File: Bsc Practicals/Prime_number.cpp
/*
 * Author : Rohit Shakya
 * Date   : June-July 2016
 *
 * Compiler : g++ 5.1.0
 * flags    : -std=c++14
 */
#include<iostream>
using namespace std;
int main()
{
	int n;
	cout<<"enter the number upto u want prime no"<<endl;
	cin>>n;
	for(int j=2;j<=n;j++)
	{
		int flag=0;
	
	for(int i=2;i<=j/2;i++)
	{
		if(j%i==0)
		flag=1;	
	}
	if(flag==0)
	cout<<j<<" ";  
   }
}

File: 2. Coding_Problems/Python's assignmemt/primenumber.py
# 
#   Author : Kajal Shakya
#   Date   : Jan-2021
#   IDE	   : Python Idle 3.7
#   Time complexity : O(n)
#   Title : Prime number
# 
num=int(input("enter a number:"))
#check for factors
flag=True
for i in range(2,num):
    if(num%i)==0:
        flag=False
        break
if(flag):
    print(num,"is a prime number")
else:
    print(num,"is not a prime number")
